### PR TITLE
OSSM-3270: add maistra proxy-update-istio postsubmit job

### PR DESCRIPTION
**Pull request by yxun**

### CODE REVIEW

**Comment by yxun** [comment]
/pj-rehearse pull-ci-maistra-proxy-maistra-2.2-copy-artifacts-gcs-2-2

**Comment by yxun** [comment]
/pj-rehearse pull-ci-maistra-proxy-maistra-2.2-proxy-update-istio-2-2

**Comment by yxun** [comment]
/pj-rehearse pull-ci-maistra-proxy-maistra-2.2-proxy-update-istio-2-2

File: ci-operator/config/maistra/proxy/maistra-proxy-maistra-2.2.yaml [hunk 1]
```diff
@@ -138,6 +138,26 @@ tests:
...
+    export ISTIO_ENVOY_BASE_URL="https://storage.googleapis.com/maistra-prow-testing/proxy"
+    git clone https://github.com/maistra/test-infra.git
+    cd test-infra
+    ./tools/automator.sh \
+      -o maistra \
+      -r istio \
+      -f /creds-github/token \
+      "-c bin/update_proxy.sh ${PULL_BASE_SHA}" \
+      '-t Automator: Update proxy' \
+      -l auto-merge \
```
Comment (jwendell) [inline ci-operator/config/maistra/proxy/maistra-proxy-maistra-2.2.yaml:153]: remove this

File: ci-operator/config/maistra/proxy/maistra-proxy-maistra-2.2.yaml [hunk 2]
```diff
@@ -138,6 +138,26 @@ tests:
...
+      -m bump-proxy \
```
Comment (jwendell) [inline ci-operator/config/maistra/proxy/maistra-proxy-maistra-2.2.yaml:154]: bump-proxy-2.2

**Pull request merged**